packs/elastic/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+st2:
```

Comment:
I don't remember if this is explicitly documented somewhere right now, but the convention we have been following so far is to put all the additional scripts and files in the `etc/` subdirectory.